test/unit/fixtures/page-block-table-in-table.html

```diff
@@ -0,0 +1,97 @@
+<main>
+    <div><h1 id="page-blocks-with-html-tables-in-tables">Page Blocks with html tables in tables</h1>
+        <div class="header-1-header-2">
```

Comment:
I think it would be more useful if multiple header values were added as separate block classes so one could inherit general CSS from a `.header-1 {}` rule in `(.header1).header-2 {}`, avoiding code duplication in an otherwise disconnected `.header-1-header-2 {}` rule.

Reply:
well, this would be a breaking change.... @davidnuescheler WDYT?

Reply:
btw, I also fixed the `toClassname` to make it nicer. but it now has errors in the diff tests:

![image](https://user-images.githubusercontent.com/917628/117945919-c5f3b600-b349-11eb-9f48-8bae6514bbc4.png)

![image](https://user-images.githubusercontent.com/917628/117946000-d9068600-b349-11eb-8cd8-de6540c8a92b.png)


@davidnuescheler is this a problem?

Reply:
i don't think i understand what `header1-header-2` even is...  would that be in a case where the first row of `headers` in a block has multiple columns? i don't think we rely on that, and i think we probably should remove support for it.
i just fixed up some the earliest blocks that used that pattern, and i don't think we have that anywhere anymore.

Reply:
the blog has some of those.... i'll keep the current behaviour.